src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static String[] result;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName));
+            try {
+                StringBuilder stringBuilder = new StringBuilder();
+                int value = bufferedReader.read();
+                while (value != -1) {
+                    stringBuilder.append((char) value);
+                    value = bufferedReader.read();
+                }
+                result = stringBuilder.toString().split("\\W+");
+
+            } catch (IOException e) {
+                throw new RuntimeException("can't read to the file" + e);
```

Comment:
```suggestion
                throw new RuntimeException("Can't read the file" + fromFileName, e);
```

let's make exception messages more informative